### PR TITLE
Fix space below footer

### PR DESCRIPTION
**Pull request by leftclickben**
Remove a JavaScript hack that was causing a large
unwanted white space below the footer.

### CODE REVIEW

**Comment by leftclickben** [comment]
This doesn't work yet, I'll close the PR.  Will reopen after further commits.

**Comment by leftclickben** [comment]
OK this is working correctly now.  The footer is always at the bottom of the page, and the left hand side nav bar always (looks) full height of the page.

File: themes/default/css/_leftNav.scss [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-	top: 0;
-	left: 0;
-	bottom: 0;
+	z-index: 2;
+	float: left;
+	height: auto;
 
 	@include breakpoint(baby-bear) {
-		
+		width: 35%;
```
Comment (kehh) [inline themes/default/css/_leftNav.scss]: Should this be on the smallest device

Comment (leftclickben) [inline themes/default/css/_leftNav.scss]: Yeah I think you're right but I think the problem is actually in the definitions of the breakpoints.

**Comment by leftclickben** [comment]
Have updated based on discussion, including changes to the breakpoint definitions.

**Pull request merged**